### DISCUSSION TITLE
Logging vs. commands

### DISCUSSION THREAD

**Discussion by blexter** [tobiasehlert/teslamateapi#214] [Q&A]
Hi,
I havnt found any instructions/documentation between the two api logging vs commands.
I get the same response if i send a GET request to both of them (with token).
Can someone explain.

BR
Andreas

**Comment by LelandSindt** [comment]
Andreas,

Can you include some examples of the commands you a trying and the responses you are getting? -- this would help me understand where you are coming from and how best to help you.

**Comment by blexter** [comment]
The response i get is (the availible commands), regardless if i call https://{url}/api/v1/cars/{carid}/command or https://{url}/api/v1/cars/{carid}/logging:

{"enabled_commands":["/command/actuate_trunk","/command/window_control","/command/auto_conditioning_start","/command/auto_conditioning_stop","/command/set_temps","/command/set_preconditioning_max","/command/remote_seat_heater_request","/command/remote_seat_cooler_request","/command/remote_steering_wheel_heater_request","/command/set_bioweapon_mode","/command/set_climate_keeper_mode","/command/remote_auto_seat_climate_request","/command/media_toggle_playback","/command/media_next_track","/command/media_prev_track","/command/media_next_fav","/command/media_prev_fav","/command/media_volume_up","/command/media_volume_down","/command/schedule_software_update","/command/cancel_software_update","/logging/resume","/logging/suspend","/command/speed_limit_set_limit","/command/speed_limit_activate","/command/speed_limit_deactivate","/command/speed_limit_clear_pin","/command/set_valet_mode","/command/reset_valet_pin","/command/charge_port_door_open","/command/charge_port_door_close","/command/charge_start","/command/charge_stop","/command/charge_standard","/command/charge_max_range","/command/set_charge_limit","/command/set_charging_amps","/command/set_scheduled_charging","/command/set_scheduled_departure","/command/upcoming_calendar_entries","/command/dashcam_save_clip","/command/honk_horn","/command/flash_lights","/command/trigger_homelink","/command/door_unlock","/command/door_lock","/command/sun_roof_control","/command/share","/command/navigation_sc_request","/wake_up","/command/remote_start_drive","/command/set_sentry_mode"]}

**Reply by LelandSindt** [reply]
Interesting.. I will see if I can recreate this as time allows.  --- can't think of why this would be happening off the top of my head.

**Comment by blexter** [comment]
I dont say its wrong, just missing documentation/instructions what/when/why use either /logging or /command (or command is documented, but how to use one or the other, different use cases and so on)

**Comment by LelandSindt** [comment]
Ah, you are looking for additional documentation and example commands.. yes?

**Comment by blexter** [comment]
Not necessary, only a short explanation should be enough, example of commands i got when i run the GET request, so its ok. But more lika above, what/when/why of the "logging".

**Comment by tobiasehlert** [comment]
@blexter in the [readme](https://github.com/tobiasehlert/teslamateapi/blob/main/README.md#commands) there is a reference to the [tesla-api.timdorr.com](https://tesla-api.timdorr.com) documentation and here you can for instance find, how to do a `POST` request to [honk the horn](https://tesla-api.timdorr.com/vehicle/commands/alerts#post-api-1-vehicles-id-command-honk_horn) of your car.

The only thing to think of is that you point to the right URL so that the TeslaMateApi catches the command and forwards it towards the owner-api.

The logging command is slightly different and thats about starting and stopping logging of TeslaMate when not using the streaming API. The functionality was added a while ago and information can be found in the issue https://github.com/tobiasehlert/teslamateapi/issues/10 in this repo. It's basically sending the `PUT` to TeslaMate instead of the owner-api by Tesla.
